trail/gallery/templates/gallery_table.html

```diff
@@ -0,0 +1,39 @@
+{% load static %}
+<!--
+         Posts
+         Loads them from the database and currently it is hard coded to display 12 images per page.
+    -->
+<div class="pagination">
+    <!--<a href="#" class="previous">Prev</a>-->
```

Comment:
If we don't need it best delete it? I'd suggest to remove the comment and resolve this comment.